carbon-design-system/carbon-addons-iot-react
# feat(barchartcard): add maximum data points field

Closes #3663 

**Summary**

- Add number input field for maximum data points

**Change List (commits, features, bugs, etc)**

- Add `maximumDataPoints` property to card editor
- Add unit test
- Update snapshots

**Acceptance Test (how to verify the PR)**

- Go to [this story](https://deploy-preview-3753--carbon-addons-iot-react.netlify.app/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-dashboardeditor-%E2%98%A2%EF%B8%8F-cardeditor--for-bar-chart)
- Open "Settings" tab
- Change value of "Maximum data points" field
- Verify applied changes in the "Action" tab

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
